src/mock/quic_p2p/mod.rs

```diff
@@ -12,6 +12,7 @@ mod node;
 mod tests;
 
 pub use self::network::Network;
+pub type Token = u64;
```

Comment:
I was thinking we could just use the real `quic_p2p`'s `Token` here, something like: `pub use quic_p2p::Token;` (should be possible, as the crate is linked unconditionally, I think, and it's not shadowed by our mock version because it's `crate::quic_p2p`). Not a blocker, though.